index.html

```diff
@@ -108,61 +108,74 @@ <h2 class="title">My skills</h2>
                 <div class="column right">
                     <div class="bars">
                         <div class="info">
-                            <span>Data Structures</span>
+                            <span>Python</span>
+                            <span>85%</span>
+                        </div>
+                        <div class="line dsl"></div>
+                    </div>
+                    <div class="bars">
+                        <div class="info">
+                            <span>Algorithm</span>
                             <span>75%</span>
                         </div>
                         <div class="line dsa"></div>
                     </div>
                     <div class="bars">
                         <div class="info">
-                            <span>Algorithm</span>
+                            <span>Data Structures</span>
                             <span>75%</span>
                         </div>
                         <div class="line dsa"></div>
                     </div>
-                        <div class="bars">
-                            <div class="info">
-                                <span>JavaScript</span>
-                                <span>70%</span>
-                            </div>
-                            <div class="line js"></div>
+                    <div class="bars">
+                        <div class="info">
+                            <span>C++</span>
+                            <span>70%</span>
                         </div>
-                        <div class="bars">
+                        <div class="line cpp"></div>
+                    </div>
+                    <div class="bars">
                             <div class="info">
-                                <span>C++</span>
+                                <span>Django Framework</span>
                                 <span>70%</span>
                             </div>
-                            <div class="line cpp"></div>
-                        </div>
-                        <div class="bars">
+                         <div class="line cpp"></div>

```

Comment:
line cpp should be changed to line django

Reply:
all right ill rename all accordingly